packages/tradershub/src/components/EUDisclaimerMessage/EUDisclaimerMessage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Text } from '@deriv/quill-design';
+
+const EUDisclaimerMessage = () => (
+    <div className='absolute w-full bottom-50 bg-system-light-secondary-background'>
+        <div className='max-w-[1232px] mx-auto px-500 lg:px-50'>
```

Comment:
the text is supposed to be full width
https://figma.com/file/Ndk2ypeMLDLkD5v0aDfbwA/Release-(Rebuild)---EU-Trader%E2%80%99s-hub?node-id=40%3A191909&mode=dev